.prettierignore

```diff
@@ -2,6 +2,10 @@
 .github/**/*.md
 .github/CODEOWNERS
 
+# Documentation
+# This file is linted when created, but the hr elements are replaced for correct rendering.
+documentation/meta/media_properties.md
```

Comment:
It looks like the linting step in the `justfile` happens _after_ the file is moved here, not before. So I think this will prevent any linting from happening at all 😞

Reply:
Fixed in [b5cd89b](https://github.com/WordPress/openverse/pull/3620/commits/b5cd89b8aa65f3612b6dc9c2a01489d5a86721d4)